### ISSUE TITLE
Audit log UI search does not search entire time range.

### ISSUE THREAD

**Issue by fspmarshall** [gravitational/teleport#33672]
Typing in a search term in the audit log UI only filters the already loaded events.  You have to manually click 'fetch more' repeatedly to get the search to be applied to the subsequent chunks.  In clusters with large numbers of events, this can mean that you often get an empty search results page and have to click 'load more' ten or more times before items start appearing.

If there are matches within the specified time range, they should be displayed (or at least the first `page_size` set of matches).

**Comment by AlgirdasEidukaitis** [comment]
Had the same experience.
We’ve had multiple users report similar concerns about this functionality. It's easy for users to misinterpret the results, assuming there are no events matching their query, and draw incorrect conclusions.

When end users search within a specified time frame, they generally expect the search to cover the entire range, not just the initial batch of loaded data. Without a clear indication that the results are incomplete, this can lead to misunderstandings.

We would love to see it included in a future release.

**Comment by pschisa** [comment]
+1 for all events in timespan when filtering to be viewed (at least the first page worth)

**Comment by zmb3** [comment]
@avatus do your recent updates address this issue?

**Comment by avatus** [comment]
yes, this will be fixed with https://github.com/gravitational/teleport/pull/60480
